pom.xml

```diff
@@ -45,7 +45,7 @@
         <java.target>1.8</java.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <mule.api.version>1.1.1</mule.api.version>
+        <mule.api.version>1.1.4-SNAPSHOT</mule.api.version>
```

Comment:
Please create a JIRA to remove the snapshot when we release.

Reply:
https://www.mulesoft.org/jira/browse/MMP-419